class-tgm-plugin-activation.php

```diff
@@ -2713,7 +2713,7 @@ protected function get_row_actions( $item ) {
 			}
 
 			$prefix = ( defined( 'WP_NETWORK_ADMIN' ) && WP_NETWORK_ADMIN ) ? 'network_admin_' : '';
-			return apply_filters( "tgmpa_{$prefix}plugin_action_links", array_filter( $action_links ), $item['slug'], $item, $this->view_context );
+			return apply_filters( 'tgmpa_' . $prefix . 'plugin_action_links', array_filter( $action_links ), $item['slug'], $item, $this->view_context );
```

Comment:
Fair point.

> Of course, we could use WP-Parser if we wanted to create some technical API documentation.

I was considering something like that at some point once all the hooks are documented. 
All the same, if the WP Parser is that finicky, we may want to consider using [WP Hook documentor](https://github.com/jrfnl/wp-hook-documentor) instead. At least it parses all variants of hooks ;-)

Or maybe better: fix WP Parser ... (yet another project to fix... *sigh*)